docs/release-notes/NuGet-5.7.md

```diff
@@ -0,0 +1,90 @@
+---
+title: NuGet 5.7 Release Notes
+description: Release notes for NuGet 5.7 including new features, bug fixes, and DCRs.
+author: chgill-msft
+ms.author: chgill
+ms.date: 8/14/2020
+ms.topic: conceptual
+---
+
+# NuGet 5.7 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
+|:---|:---|:---|
+| [**5.7.0**](https://nuget.org/downloads) | [Visual Studio 2019 version 16.7](https://visualstudio.microsoft.com/downloads/) | [3.1.401](https://dotnet.microsoft.com/download/dotnet-core/3.1)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio 2019 with .NET Core workload
+
+## Summary: What's New in 5.7
+
+### Features added in this release
```

Comment:
Probably not for this release, but we can make these titles a bit more user friendly in 5.8+. For example, `Added Visual Studio partial restore for PackageReference projects (no-op++)` might be something we could reword to talk about the benefits that partial restore (i.e. no-op) provides to PackageReference projects.

Reply:
I agree with this 😊